lms/templates/courseware/accordion.html

```diff
@@ -40,9 +40,12 @@
                 ## a status icon
                 <%
                     if section.get('due') is None:
-                        data_string = ''
+                        data_string = _("{section_format}").format(section_format=section['format'])
```

Comment:
Thanks for iterating on this, Greg. Given that there's nothing in this string except a placeholder, there is no need to use "_" for translation.

To see where this value is specified, look at Studio, Settings, Grading.